### DISCUSSION TITLE
Scalibility

### DISCUSSION THREAD

**Discussion by aarnavpbhat** [pocketbase/pocketbase#2172] [Q&A]
How many people can Pocketbase handle on a scale?

**Comment by Xyphyn** [comment]
Quoting the docs,

> Even without optimizations, PocketBase can easily serve 10 000+ concurrent and persistent realtime connections on a cheap $6 VPS (Hetzner 2vCPU, 4GB RAM).

The answer is, it depends. You probably won't be able to host something on the scale of Twitter or Facebook, but it will certainly scale well enough for many projects.
